enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,172 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-27
+last-updated: 2022-08-25
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster admins can provide additional catalog items and types by installing operators or adding helm repositories.
+Cluster administrators want to hide these types.
+
+Under the developer catalog,
+there are categories and sub-categories. When users select a sub-category, they can see the list
+of types to install. This enhancement allows cluster admins to selectively disable these types.
+
+![Developer catalog and types list](customize-catalogs.png)
+ 
+## Motivation
+
+Cluster administrators don't want the users to use the catalog 
+to install anything in the cluster until
+customization. Cluster administrators want to 
```

Comment:
I'm a little unclear on this first sentence... Nothing is blocking the user from doing this, correct?

Reply:
Updated the doc. PTAL.